common/dynamicconfig/constants.go

```diff
@@ -673,6 +673,41 @@ const (
 	// TransferQueueMaxReaderCount is the max number of readers in one multi-cursor transfer queue
 	TransferQueueMaxReaderCount = "history.transferQueueMaxReaderCount"
 
+	// CallbackTaskBatchSize is batch size for callbackQueueFactory
+	CallbackTaskBatchSize = "history.callbackTaskBatchSize"
+	// CallbackProcessorFailoverMaxPollRPS is max poll rate per second for callbackQueueFactory
+	CallbackProcessorFailoverMaxPollRPS = "history.callbackProcessorFailoverMaxPollRPS"
+	// CallbackProcessorMaxPollRPS is max poll rate per second for callbackQueueFactory
+	CallbackProcessorMaxPollRPS = "history.callbackProcessorMaxPollRPS"
+	// CallbackProcessorMaxPollHostRPS is max poll rate per second for all callbackQueueFactory on a host
+	CallbackProcessorMaxPollHostRPS = "history.callbackProcessorMaxPollHostRPS"
+	// CallbackProcessorSchedulerWorkerCount is the number of workers in the host level task scheduler for callbackQueueFactory
+	CallbackProcessorSchedulerWorkerCount = "history.callbackProcessorSchedulerWorkerCount"
+	// CallbackProcessorSchedulerActiveRoundRobinWeights is the priority round robin weights used by callback task scheduler for active namespaces
+	CallbackProcessorSchedulerActiveRoundRobinWeights = "history.callbackProcessorSchedulerActiveRoundRobinWeights"
```

Comment:
I think a more consistent name would be either `CallbackProcessorActiveSchedulerRoundRobinWeights` or `CallbackProcessorSchedulerRoundRobinWeightsActive`. I prefer `ActiveScheduler`. The reason is that there is no `CallbackProcessorSccheduler` setting or struct here. So, I would go with proper English.

Reply:
I agree we could use a review for the dynamic config structure and naming convention.
I'm just going with the same convention we have for the other processors.
If we want to change the naming, we should change all names and that should probably be in a PR that's not part of the feature branch.